pkg/pillar/cmd/logmanager/logmanager.go

```diff
@@ -692,11 +692,21 @@ func sendProtoStrForLogs(reportLogs *logs.LogBundle, image string,
 	}
 	size := int64(proto.Size(reportLogs))
 	if size > logMaxBytes {
-		log.Warnf("sendProtoStrForLogs: %d bytes: %s\n",
-			size, reportLogs)
+		log.Warnf("LogBundle Start: DevID %s, Image %s, EveVersion %s",
+			reportLogs.DevID, reportLogs.Image, reportLogs.EveVersion)
+		for _, logEntry := range reportLogs.Log {
+			log.Warnf("----> %s", logEntry)
+		}
```

Comment:
I suggest we drop this loop and anything else which logs the entries; that reduces the risk of an inifite log generator should something happen where logmanager ends up processing its own log entries (I know that shouldn't happen just like nuclear reactors should not overheat; I want a guard when and if it does happen).

Since we gather multiple log entries into one bundle it should be ok to log for each bundle even if there is a loop since multiple entries go into one bundle (hence the feedback loop will have a gain far less than one).

Reply:
will do